exchanges/gateio/risk_test.go

```diff
@@ -0,0 +1,161 @@
+package gateio
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/sharedtestvalues"
+	testexch "github.com/thrasher-corp/gocryptotrader/internal/testing/exchange"
+)
+
+func TestGetUnifiedUserRiskUnitDetails(t *testing.T) {
+	t.Parallel()
+
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, e)
+
+	got, err := e.GetUnifiedUserRiskUnitDetails(t.Context())
+	require.NoError(t, err)
+	assert.NotEmpty(t, got)
+}
+
+func TestGetFuturesRiskTable(t *testing.T) {
+	t.Parallel()
+
+	_, err := e.GetFuturesRiskTable(t.Context(), currency.EMPTYCODE, "")
+	require.ErrorIs(t, err, currency.ErrCurrencyCodeEmpty)
+
+	_, err = e.GetFuturesRiskTable(t.Context(), currency.USDT, "")
+	require.ErrorIs(t, err, errTableIDEmpty)
+
+	got, err := e.GetFuturesRiskTable(t.Context(), currency.USDT, "CYBER_USDT_20241122")
```

Comment:
🤷‍♂️ Clarification please: Can we really rely on this contact being here?
We're not mocking here.
Should we get a contract randomly to be sure?

Reply:
Yeah I think you are correct, will change this.

Reply:
https://github.com/thrasher-corp/gocryptotrader/pull/2106/commits/e0a9bcdb6a325839d9c2b61e139a961134c1ef5c